auth/auth.go

```diff
@@ -0,0 +1,27 @@
+package auth
```

Comment:
Add a note about the package being unstable and subject to breaking changes, whether it's here or in a `auth/docs.go` file.